src/views/documents/utils/QuerySortDropdown.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <dropdown-button>
+    <span slot="button" class="button is-small-mobile" :disabled="false">
+      <fa-icon icon="sort" />
```

Comment:
I would hide this icon when text is displayed, I find it rather confusing because it could also by used as an indicator for a dropdown input, thus we have it twice, somehow. You understand it on mobile, because all buttons have no text. But on desktop, I thought it was a bug first.

Not sure I'm totally clear?

Reply:
You are clear. 

But what is your proposition? 

1. Remove the icon? then we should remove it on other dropdown button, it would be strange to have only one button without icon, no? 
2. find a better icon ? Probably, I've been lazy on that one, just took the font-awesome `sort` icon 😅

Reply:
3. Keep the icon on mobile, hide in desktop (text suffice).

Option 2 could be ok too.

Reply:
I've changed the icon :  ![image](https://user-images.githubusercontent.com/11915659/122797496-d0498e00-d2bf-11eb-8fd0-40cc121d1aef.png)